src/transformations/utils/build-super-calls-for-chain.ts

```diff
@@ -110,6 +109,32 @@ function isImplicitlyConstructed(contract: ContractDefinition): boolean {
 
   return (
     contract.contractKind === 'contract' &&
-    (ctor == undefined || ctor.parameters.parameters.length === 0)
+    (ctor == undefined ||
+      ctor.parameters.parameters.length === 0 )
+  );
+}
```

Comment:
This was only modified by the linter (i guess), but it makes me wonder. Should there be a case about the modifier.

`constructor() mySideEffectModifier() {}` is not implicitly constructed.

Reply:
or maybe in that case the contract will not be `isEmpty`, and that is good enough